namer/web/routes/api.py

```diff
@@ -13,13 +13,13 @@
 from namer.web.actions import delete_file, get_failed_files, get_queue_size, get_queued_files, get_search_results, read_failed_log_file
 
 
-def get_web_api(config: NamerConfig, command_queue: Queue) -> Blueprint:
+def get_routes(config: NamerConfig, command_queue: Queue) -> Blueprint:
     """
     Builds a blueprint for flask with passed in context, the NamerConfig.
     """
-    blueprint = Blueprint('api', __name__, static_url_path='/', static_folder='public', template_folder='templates')
+    blueprint = Blueprint('api', __name__, url_prefix='/api')
```

Comment:
concern is this could break the webroot, but maybe not, have you tested with a webroot?